BrowserKit/Sources/SummarizeKit/Backend/LLM/LiteLLMClient.swift

```diff
@@ -21,7 +21,7 @@ final class LiteLLMClient: LiteLLMClientProtocol, @unchecked Sendable {
     init(
         apiKey: String,
         baseURL: URL,
-        urlSession: URLSessionProtocol = URLSession.shared
+        urlSession: URLSession = URLSession.shared
```

Comment:
This was changed because we moved all the classes to `BrowserKit` and we don't have `URLSessionProtocol`. We don't need that though for testing we can use something similar to this https://github.com/mozilla-mobile/firefox-ios/blob/231a8bf7d85f327336265255e3a5361574a0b907/firefox-ios/firefox-ios-tests/Tests/ClientTests/TemporaryDocumentTests.swift#L16-L25